docs/csharp/deconstruct.md

```diff
@@ -45,11 +45,17 @@ There are two ways to deconstruct a tuple:
 
     This is cumbersome and is not recommended.
 
+- Lastly, you may deconstruct the typle into variables that have already been declared.
```

Comment:
'typle' should be changed to 'tuple'

Reply:
Oops! Thanks for pointing that out! Fixed now.